fabfile.py

```diff
@@ -168,6 +168,33 @@ def run_migrations():
         warn("Refusing to run migrations in staging environment")
 
 
+@task
+def build_measures(environment=None, measures=None):
+    env.app = environments[environment]
+    env.environment = environment
+    env.path = "/webapps/%s" % env.app
```

Comment:
Could you extract this code to set up the environment as its own function, which you could then also call from `deploy()`?

And while I'm looking at this -- should `graceful_reload` also take an environment?

Reply:
1. Yes
2. No - I see no benefit to it being its own fab command so instead think we should remove the `@task`